tools/EteriTools/CollectResults/evaluation.py

```diff
@@ -0,0 +1,114 @@
+import glob, os, sys
+import shutil
+from shutil import copyfile
+from os import system
+import re
+#import sklearn
+#from sklearn import metrics
+#from tabulate import tabulate
+
+
+
+def sh(script):
+    system("bash -c '%s'" % script)
+
+#dataNames = "test-data/FASTA/data.names"
+dataNames = "FASTA/data.names"
+#path = sys.argv[2]
+
+
+listOfClusters = []
+listOfClasses = []
+
+#cluster_seqs_stats_path = "test-data/RESULTS/*.cluster.all"
+
+cluster_seqs_stats_path = "RESULTS/*.cluster.all"
+cluster_seqs_stats_files = glob.glob(cluster_seqs_stats_path)
+
+blackList =[]
+numberOfClusters = 0
+for singleFile in sorted(cluster_seqs_stats_files):
+    numberOfClusters += 1
+    with open(singleFile, "r") as f:
+        for line in f.readlines():
+            uniqueId = line.split()[6] #was 8
+            print "unique id = ", uniqueId
+            clustNum = line.split()[1]
+            rnaClass, sep, tail = uniqueId.partition("_")
+            #classToClusterDict[rnaClass] = clustNum
+            listOfClasses.append(rnaClass)
+            listOfClusters.append(clustNum)
+            print "rnaClass = ", rnaClass
```

Comment:
Can you use logging if you want to produce debug output or remove this in the final script.